cogs/embeds.py

```diff
@@ -300,13 +300,7 @@ async def transfer(self, ctx, amount: int, member: discord.Member):
             await ctx.channel.send(
                 "Provided amount is higher than owned credits.")
             return
-        dbprof_member = await db.get_member(member.guild.id, member.id)
-        await db.set_member(
-            dbprof_author['guild_id'], dbprof_author['member_id'], 'cash',
-            dbprof_author['cash'] - amount)
-        await db.set_member(
-            dbprof_member['guild_id'], dbprof_member['member_id'], 'cash',
-            dbprof_member['cash'] + amount)
+        await db.transfer_currency(ctx.guild.id, ctx.author.id, member.id, amount)
         await ctx.channel.send(
             f"{member.mention} recieved {amount} credit(s) from {ctx.author.mention}.")
```

Comment:
I would like to see this string rephrased to `{ctx.author.mention} has given {amount} credits to {member.mention}.`